src/main/resources/library/attribute-definitions.json

```diff
@@ -257,7 +265,8 @@
         {"property": "library:broadInternalResearchProjectID"},
         {"property": "library:broadInternalResearchProjectName"},
         {"property": "library:broadInternalCohortName"},
-        {"property": "library:broadInternalSeqProjectNumbers", "inputHint": "Broad internal IDs"}
+        {"property": "library:broadInternalSeqProjectNumbers", "inputHint": "Broad internal IDs"},
+        {"property": "library:lmsvn", "editable": false}
```

Comment:
yeah, I don't really like the duplication that's there now either, but thanks for this fix! PR looks good to me 👍